fastlane/Fastfile

```diff
@@ -1,17 +1,17 @@
-default_platform(:ios)
+default_platform(:mac)
 
-platform :macos do
+platform :mac do
```

Comment:
Why is this change needed? Is it because of bitrise?
If it is needed, you will have to update the README file:
`Run fastlane macos ci` => `Run fastlane mac ci`

Reply:
There's no such platform as "macos". With this change you can simply call `fastlane ci`.
But yeah, I need to update README